django_dramatiq/middleware.py

```diff
@@ -46,6 +47,7 @@ def after_process_message(self, broker, message, *, result=None, exception=None,
 
         if exception is not None:
             status = Task.STATUS_FAILED
+            message.options["traceback"] = traceback.format_exc(limit=30)
```

Comment:
format_exc require to be inside try/except, i think we can replace it by more "durable" solution

```python
            message.options['traceback'] = ''.join(
                traceback.format_exception(
                    exception,
                    limit=30,
                )
            )
```

Reply:
Thanks for the suggestion! 👍